packages/tsconfig-reference/copy/en/options/erasableSyntaxOnly.md

```diff
@@ -0,0 +1,62 @@
+---
+display: "Erasable Syntax Only"
+oneline: "Do not allow runtime constructs that are not part of ECMAScript."
+---
+
+Node.js [supports running TypeScript files directly](https://nodejs.org/api/typescript.html#type-stripping) as of v23.6;
+however, only TypeScript-specific syntax that does not have runtime semantics are supported under this mode.
+In other words, it must be possible to easily *erase* any TypeScript-specific syntax from a file, leaving behind a valid JavaScript file.
+
+That means the following constructs are not supported:
```

Comment:
We might want to wait to merge this until https://github.com/microsoft/TypeScript/pull/61244 (which I believe we're going to talk about in a meeting soon).